pairAndIntroduce.ts

```diff
@@ -1,11 +1,13 @@
 import { App } from '@slack/bolt';
+import { findConversation } from './utils/buttons';
 
 require('dotenv').config();
-
 // Initializes your app with your bot token and signing secret
 const app = new App({
   token: process.env.SLACK_BOT_TOKEN,
   signingSecret: process.env.SLACK_SIGNING_SECRET,
   socketMode: true, // add this
   appToken: process.env.SLACK_APP_TOKEN // add this
-});
\ No newline at end of file
+});
+
+findConversation(app).catch((e) => console.error(e['data']));
```

Comment:
We definitely don't want this to be called every time we run the app depending on the situation, but I'm not sure where this function would be called? Maybe we need a switch on `startConversations` vs `findConversations`